examples/run-blockchain/index.ts

```diff
@@ -0,0 +1,118 @@
+import VM from '../../'
+
+import Account from 'ethereumjs-account'
+import * as utils from 'ethereumjs-util'
+import { promisify } from 'util'
+
+const Trie = require('merkle-patricia-tree/secure')
+const Block = require('ethereumjs-block')
+const Blockchain = require('ethereumjs-blockchain')
+const BlockHeader = require('ethereumjs-block/header.js')
+const level = require('level')
+const levelMem = require('level-mem')
+const testData = require('./test-data')
+
+const rlp = utils.rlp
+
+async function main() {
+  const state = new Trie()
+  const blockchainDB = levelMem()
+
+  const hardfork = testData.network.toLowerCase()
+
+  const blockchain = new Blockchain({ db: blockchainDB, hardfork })
+  blockchain.ethash.cacheDB = level('./.cachedb')
```

Comment:
I'm not sure what this line does. Is this different from the `blockchainDB`?

Reply:
Yeah, I tested the same thing, got the same results. I guess, given that it's an ethash thing, that it only matters if you are validating PoW.

Will remove it.

Reply:
Re-tested this, and it does change the performance. I reintroduced it with some comments about it.